stockflux-securities-master/src/components/ToggleSwitch.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./ToggleSwitch.css";
+
+const ToggleSwitch = props => {
+  return (
+    <>
+      <input
+        {...props}
+        type="checkbox"
+        className={"input-form-toggle" + ` ${props.className}`}
```

Comment:
Similar to in the `TextField` component here, if you expect these properties in the component arguments directly, you can remove the `props.` prefix since they will exist as their own variables. You can also use the backtick, string interpolation feature to simply the `className` expression:

    className={`input-form-toggle ${className}`}

Reply:
Yep I've done this.

Worth noting:  I've moved the extra className from the input to the label as the input is hidden so extra css won'yt do anything to it. The label is the one which is actually displayed

Reply:
Cool. Yeah, that's the right approach since the input is hidden anyway.